paper/Makefile

```diff
@@ -0,0 +1,10 @@
+all : paper.pdf
+
+paper.pdf : paper.md paper.bib example-modular-augur-workflows.pdf
+	pandoc --citeproc --bibliography=paper.bib --template template.tex -s $< -o $@
```

Comment:
I'm getting an error when running this line that `Could not find data file templates/template.tex`. I don't see `template.tex` as part of `paper/` in this repo. Was this supposed to ship with pandoc? I'm using brew installed pandoc rather than conda installed pandoc (which could be the source of the error).

Reply:
Thank you for catching this. I just added the missing `template.tex` file. This is nearly identical to the default pandoc LaTeX template (`pandoc -D latex > template.tex`), but it updates the authors/affiliations blocks to work with JOSS's formatting expectations.

Reply:
This resolved the issue for me. PDF build now works. Thanks!